lisp/Makefile

```diff
@@ -24,16 +24,17 @@ NOWEAVE=noweave
 COMPILE = $(EMACSBATCH) --directory . -f batch-byte-compile
 
 ELS = $(wildcard *.el)
+ELS := $(filter-out ess-autoloads.el, $(ELS))
 ELC = $(ELS:.el=.elc)
 
 # ESSR_VER =`cat ../etc/ESSR-VERSION`
 ##ESSR_VER = $(shell cat ../etc/ESSR-VERSION)
 
 ### Targets
 
-all: julia-mode.el $(ELC) ess-custom.el
+all: julia-mode.el $(ELC) ess-custom.el ess-autoloads.el
 
-.PHONY: dist
+.PHONY: dist ess-autoloads.el
```

Comment:
Why did you make it a  .PHONY target? It's a real file.

Reply:
It also probably should not depend on ELCs either because els are scanned for cookies.

Reply:
It started out as a phony target and I forgot to change it. I've deleted that and the elcs dependency.

Reply:
Oh, you did that. Nevermind :-)